src/components/inputs/z-input/styles-text.css

```diff
@@ -7,6 +7,7 @@
   width: 100%;
   margin: 0;
   padding: 0 var(--half-x3);
+  padding-right: calc(3 * var(--half-x3));
```

Comment:
We should avoid overridden values inside the same rule 😅 . Furthermore, we have the `--half-x9` variable. May I suggest `padding: 0 var(--half-x9) 0 var(--half-x3);`?

Reply:
Sorry I have missed the 'x9' property. Yes, totally agree, I will use this rule you proposed.